assets/src/scss/editorStyle.scss

```diff
@@ -45,6 +45,8 @@
 @import "blocks/Submenu/SubmenuEditorStyle";
 @import "blocks/TakeActionBoxout/TakeActionBoxoutEditorStyle";
 @import "blocks/Timeline/TimelineEditorStyle";
+@import "blocks/PostsList/PostsListEditor";
+@import "blocks/ActionsList/ActionsListEditor";
```

Comment:
For consistency, these files should be named `ActionsListEditorStyle` and `PostsListEditorStyle`

Reply:
Updated!